### PR TITLE
a hidden field for the registration form

### PR DESCRIPTION
**Pull request by tobiasd**
This PR adds a hidden input field to the registration form.

addresses #7688

### CODE REVIEW

File: src/Module/Register.php [hunk 1]
```diff
@@ -261,6 +261,11 @@ public static function post()
 
 				$a->internalRedirect('register/');
 			}
+			// Is there text in the tar pid?
```
Comment (AlfredSK) [inline src/Module/Register.php]: tar pit, please

File: src/Module/Register.php [hunk 2]
```diff
@@ -261,6 +261,11 @@ public static function post()
...
+			if (!empty($_POST('registeertarpid'))) {
```
Comment (annando) [inline src/Module/Register.php]: `registertarpid`

**Comment by MrPetovan** [comment]
Hopefully it works 🤞

**Comment by annando** [comment]
Why have you closed it?

**Comment by tobiasd** [comment]
Silly user deleting a ton of remote branches in his github repository I guess...

**Comment by nupplaphil** [comment]
tbh .. what does this change do? I don't get it ^^

**Comment by tobiasd** [comment]
There is an input field in the register form which is not shown in the browser. But bots will (hopefully) see it and enter some text. When text was entered, the registration process will stop.

**Comment by nupplaphil** [comment]
aaaaaahhh got it :)

**Comment by AlfredSK** [comment]
Has someone tried a registration now? Does it work? I'm curious.

**Pull request merged**